setup.cfg

```diff
@@ -74,6 +74,7 @@ disable =
   # library specific
   too-many-instance-attributes,
   too-many-arguments,
+  too-many-positional-arguments,
```

Comment:
It would be nice to synchronize this file with our template instead of just adding this one line 😉